src/types.ts

```diff
@@ -99,3 +99,8 @@ export type TaskGroupResult = {
   groupId: string;
   map: ResultMap;
 };
+
+export enum AllowedGroup {
```

Comment:
This is nicer for the TS folk, but in JS land, people would have to pass in a bare string. Is it worth providing an export for JS? Something like:

```javascript
export const AllowedGroup = Object.freeze({
  Client: 'Client',
  Server: 'Server',
});
```

or are we fine documenting the strings as-is?